lib/elixir/test/elixir/float_test.exs

```diff
@@ -104,9 +104,9 @@ defmodule FloatTest do
     assert Float.ceil(7.5432e3) === 7544.0
     assert Float.ceil(7.5e-3) === 1.0
     assert Float.ceil(-12.32453e4) === -123_245.0
-    assert Float.ceil(-12.32453e-10) === 0.0
+    assert Float.ceil(-12.32453e-10) === -0.0
```

Comment:
Was already the case on OTP26, the test was misleading precisely because `0.0 === -0.0` was true